community/0006-governance.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+This document is intended to detail the governing structure of the Quilt project.
+
+# Motivation
+This document has been written with the explicit purpose of defining guidelines and expectations for project staff. It has been written with the idea of decentralizing power in mind, to help contribute to the goal of a truly open project.
+
+# Explanation
+
+## Governmental Structure
+![Government Structure](government-structure.png)
+\* For illustration purposes only, technical teams may vary
```

Comment:
Could I ask for this image to be replaced with a more theme-friendly (both dark and light) version? I have one such version made here:

<details>
<summary><strong>Governance Structure Illustration</strong> (1403px by 992px, 120 DPI)</summary>
(If necessary, I can provide the SVG source. For licensing: I release this under the CC0)
<img src="https://user-images.githubusercontent.com/21304337/115828671-9b949400-a440-11eb-89c5-c2c282bf7722.png">
</details>

Reply:
FYI the terminology for QSL is incorrect--the things that will have teams managing them are called "libraries" which are groups of modules.

Reply:
@sciwhiz12  if you can provide a corrected diagram with the word change, please do -- but I have pushed the new image

Reply:
I based off the diagram from the original diagram from the RFC, so copied the incorrect terminology from there too. @ToffeeMax Here's a corrected version (changing `QSL Module` to `QSL Library`).

<details>
<summary><strong>Governance Structure Illustration<strong> [<em>corrected</em>] (1403px by 992px, 120 DPI)</summary>
(Licensing: under the CC0)
<img src="https://user-images.githubusercontent.com/21304337/115922963-206ac680-a4b0-11eb-9c07-ac43fa1ebda2.png">
</details>